client/apps/edit/components/section_list/index.coffee

```diff
@@ -26,6 +26,31 @@ module.exports = React.createClass
   onNewSection: (section) ->
     @setState editingIndex: @props.sections.indexOf section
 
+  convertImages: (section) ->
+    newSection = {
+      type: 'image_collection'
+      layout: section.get('layout') || 'overflow_fillwidth'
+      images: [ {
+        type: section.get('type')
+        url: section.get('url')
+        caption: section.get('caption')
+      } ]
+    }
+    section.attributes = newSection
+    return section
+
+  convertArtworks: (section) ->
+    artworks = section.get('artworks').map (artwork, i) ->
+      artwork.type = 'artwork'
+      return artwork
+    newSection = {
+      type: 'image_collection'
+      layout: section.get('layout') || 'overflow_fillwidth'
+      images: artworks
+    }
+    section.attributes = newSection
```

Comment:
So one minor thing, I think we should leverage Backbone methods here instead of setting `attributes` directly! Same with the `convertImages` part too.